docs/reference/commandline/stats.md

```diff
@@ -36,6 +36,8 @@ If you want more detailed information about a container's resource usage, use th
 
 > **Note**: On Linux, the Docker CLI reports memory usage by subtracting page cache usage from the total memory usage. The API does not perform such a calculation but rather provides the total memory usage and the amount from the page cache so that clients can use the data as needed.
 
+> **Note**: PIDS count contains the processes and kernel threads created by that process. Depending on OS, threads can be called light-weight process, kernel tasks, etc. A large number of PIDS and a small number processes (reported by `ps` or `top`) means that one process is creating lots of threads.
```

Comment:
One more note... we may want to generalize this a bit more.

> PIDS count shows the number of processes running in the container. Depending on the platform, this number can mean different things. On Linux, the number refers to the number of "tasks" running in the container where a "task" can be a process or even a thread. A large PIDs count with a small number of processes may mean that are large number of threads are running.

Reply:
> may mean that are large number of threads are running.
I think a better wording would be : "may mean that there is a large number of threads running". 

I'm ok with your proposal. Make the change if you like. Anything is better than nothing and we can go around discussing a lot with very little extra value. I think both fragments convey the idea. +1 for merging.